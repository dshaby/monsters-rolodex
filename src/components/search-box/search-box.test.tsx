import {render} from "@testing-library/react";
import SearchBox from "./search-box.component";

test("Card-List Component", () => {
    const {asFragment} = render(<SearchBox className="Sample Class" placeholder="Hey" value="me" onChangeHandler={jest.fn()} />);
    expect(asFragment()).toMatchSnapshot();
});