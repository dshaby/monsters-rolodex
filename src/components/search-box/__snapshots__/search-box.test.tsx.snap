// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card-List Component 1`] = `
<DocumentFragment>
  <div>
    <input
      class="search-box Sample Class"
      placeholder="Hey"
      type="search"
      value="me"
    />
  </div>
</DocumentFragment>
`;
